app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java

```diff
@@ -0,0 +1,65 @@
+package org.fossasia.openevent.adapters;
+
+import android.graphics.Color;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.data.Event;
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.utils.ViewHolder;
+
+import java.util.List;
+
+/**
+ * Created by md91829 on 3/3/16.
+ */
+public class ScheduleListAdapter extends BaseRVAdapter<Event,ViewHolder.Viewholder> {
+
+    public ScheduleListAdapter(List<Event> eventsList) {
+        super(eventsList);
+    }
+
+    private ViewHolder.SetOnClickListener listener;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/triplez/open-event-android/file/2406791376/issues/source?bid=3233850&fileBranchId=3233907#l27)